tests/config/run-format-test.js

```diff
@@ -78,11 +80,43 @@ const meriyahDisabledTests = new Set([
     "js/regex/regexp-modifiers.js",
   ].map((file) => path.join(__dirname, "../format", file)),
 ]);
-const babelTsDisabledTest = new Set(
+const babelTsDisabledTests = new Set(
   ["conformance/types/moduleDeclaration/kind-detection.ts"].map((file) =>
     path.join(__dirname, "../format/typescript", file),
   ),
 );
+const oxcDisabledTests = new Set([
+  ...babelOnlyTests,
+  ...[
+    // Missing `.phase`
+    // https://github.com/oxc-project/oxc/issues/10978
+    "js/babel-plugins/deferred-import-evaluation.js",
+    "js/babel-plugins/source-phase-imports.js",
+    "js/deferred-import-evaluation/dynamic-import-attributes-expression.js",
+    "js/deferred-import-evaluation/dynamic-import.js",
+    "js/deferred-import-evaluation/import-defer-attributes-declaration.js",
+    "js/deferred-import-evaluation/import-defer.js",
+    "js/dynamic-import/import-phase.js",
+    "js/source-phase-imports/import-source-attributes-expression.js",
+    "js/source-phase-imports/import-source-dynamic-import.js",
+    "js/source-phase-imports/import-source-attributes-declaration.js",
+    "js/source-phase-imports/import-source-binding-from.js",
+    "js/source-phase-imports/import-source-binding-source.js",
+    "js/source-phase-imports/import-source.js",
+
+    // Missing `.decorators`
+    // https://github.com/oxc-project/oxc/issues/10921
+    "js/babel-plugins/decorators.js",
+    "js/decorators",
+    "js/decorators/class-expression",
+    "js/decorators-export",
+    "js/decorator-auto-accessors",
+    "js/ignore/class-expression-decorator.js",
+
+    // https://github.com/oxc-project/oxc/issues/10980
+    "js/top-level-await/test.cjs", // Parses as `module` even I already told it's `script`
```

Comment:
FYI: This seems resolved now by https://github.com/oxc-project/oxc/pull/11060

Reply:
I'm aware, but we will not ship this PR until all these issues are fixed, so we will update oxc-parser in a follow-up PR.